src/main/daml/Daml/Finance/Interface/Holding/Util.daml

```diff
@@ -4,9 +4,32 @@
 module Daml.Finance.Interface.Holding.Util where
 
 import Daml.Finance.Interface.Holding.Base qualified as Base (I)
+import Daml.Finance.Interface.Holding.Fungible qualified as Fungible (I)
+import Daml.Finance.Interface.Holding.Transferable qualified as Transferable (I)
 import Daml.Finance.Interface.Types.Common.Types (InstrumentKey, Parties)
+import Daml.Finance.Interface.Util.Common (isInstanceOf)
 import Daml.Finance.Interface.Util.Disclosure qualified as Disclosure (AddObservers(..), I, RemoveObservers(..))
 
+-- | Type of holdings that can be settled as part of a settlement batch.
```

Comment:
i am slightly concerned with adding the `HoldingStandard` inside an interface package, as it is just used as a utility function inside implementation packages.

`Daml.Finance.Util` seems the right place for this type, what do you think?

Reply:
as agreed, I will keep the holding standard in the interface package (as it relates to which interfaces of that package that are implemented)